samtranslator/model/sam_resources.py

```diff
@@ -2178,10 +2199,19 @@ def to_cloudformation(self, **kwargs: Any) -> List[Resource]:
 
         if model.Functions:
             function_configurations = self._construct_appsync_function_configurations(
-                model.Functions, model.DynamoDBDataSources, model.ResolverCodeSettings, api_id
```

Comment:
why is `model.DynamoDBDataSources` taken out? Is it part of the ResolverCodeSettings now?

Reply:
There's a class variable we use now defined in the `__init__` method called `_datasource_name_map`. We use this for our reference to datasources instead of passing around the model since we were using it in multiple places and it was tedious.